Rex MiXiT côté orga (#340)
Elao/elao_: content/blog/elao/rex-orga-mixit-2021.md

---

> title:              "MiXiT 2021, une conférence 100 % en ligne: retour d'expérience côté organisation"

Author: alors je l'ai enlevé parce que sur mon écran, le titre se coupe juste à l'espace avant les ":" ça me barbe

> title:              "MiXiT 2021, une conférence 100 % en ligne: retour d'expérience côté organisation"

Reviewer: Tu peux utiliser une espace insécable pour ça ? (alt  + space)

date:               "2021-05-25"
lastModified:       ~
tableOfContent:     2

description:        "Comme la plupart des conférences tech, cette année, l'édition de MiXiT 2021 a eu lieu à distance. Alors, comment ça s'est passé côté orga ?"

thumbnail:          images/posts/2021/mixit/mixit-2021.png
tags:               ["conference", "mixit"]
categories:         ["Elao"]
authors:            ["aldeboissieu"]
---

> Le challenge de l'édition 2019, c'était le changement de lieu pour la Manufacture de Tabacs, dans le but d'accueillir plus participants : de 350 à plus de 1000. Puis, vient 2020. L'organisation se passe bien, le programme est ficelé. Mais comme vous le savez, la mort dans l'âme, nous avons dû annuler car l'incertitude de la suite pesait trop sur notre moral. Finalement, nous avons bien fait, puisque nous avons été ensuite confinés. **Finalement, l'édition 2021 nous promettait encore plus de surprises !**

Reviewer: ```suggestion
Le challenge de l'édition 2019, c'était le changement de lieu pour la Manufacture de Tabacs, dans le but d'accueillir plus participants : de 350 à plus de 1000. Puis, vient 2020. L'organisation se passe bien, le programme est ficelé. Mais comme vous le savez, la mort dans l'âme, nous avons dû annuler car l'incertitude de la suite pesait trop sur notre moral. Finalement, nous avons bien fait, puisque nous avons été ensuite confinés. **Cependant, l'édition 2021 nous promettait encore plus de surprises !**
```

> Le challenge de l'édition 2019, c'était le changement de lieu pour la Manufacture de Tabacs, dans le but d'accueillir plus participants : de 350 à plus de 1000. Puis, vient 2020. L'organisation se passe bien, le programme est ficelé. Mais comme vous le savez, la mort dans l'âme, nous avons dû annuler car l'incertitude de la suite pesait trop sur notre moral. Finalement, nous avons bien fait, puisque nous avons été ensuite confinés par la suite. **Finalement, l'édition 2021 nous promettait encore plus de surprises !**

Reviewer 2: ```suggestion
Le challenge de l'édition 2019, c'était le changement de lieu pour la Manufacture de Tabacs, dans le but d'accueillir plus participants : de 350 à plus de 1000. Puis, vient 2020. L'organisation se passe bien, le programme est ficelé. Mais comme vous le savez, la mort dans l'âme, nous avons dû annuler car l'incertitude de la suite pesait trop sur notre moral. Finalement, nous avons bien fait, puisque nous avons été ensuite confinés. **Finalement, l'édition 2021 nous promettait encore plus de surprises !**
```

Reviewer: Attention à la répétition de "Finalement"

<figure>
    <img src="images/posts/2021/mixit/mixit-fun.png">
    <figcaption>
      <span class="figure__legend">MiXiT, dans le monde d'avant.</span>
    </figcaption>
</figure>

Pourquoi faire un retour d'expérience cette année, et pas les années précédentes ? **Et bien tout simplement parce que cette année n'a pas été comme les autres sur l'organisation même de la conférence et celle de l'équipe.**

En passant, je tiens à remercier mon employeur Elao (qui héberge d'ailleurs ce billet de blog), qui permet à ses collaborateurs (et donc à moi) de prendre du temps de travail pour contribuer à des projets open source ou à des projets comme MiXiT, en estimant que cela est vertueux pour la communauté. Si vous pratiquez une activité bénévole, vous devez vous douter que c'est un filet de sécurité très confortable…

## Le choix de la conférence en ligne

> Nous décidons, en septembre 2020, d'organiser une conférence en ligne et non en présentiel, car nous savons que l'incertitude est trop démotivante pour l'équipe. Nous ne voulions pas nous lancer dans une édition "comme d'habitude mais en ligne", car nous-même n'aurions pas envie d'y participer. Nous avons plutôt imaginé **des sessions courtes, en fin de soirée, pour que les spectateurs puissent se retrouver** ensemble boire un verre de jus de pomme (ou autre) et grignoter une crêpe (ou autre), si cela pouvait être possible. Rappelez-vous, en septembre 2020: c'était difficile de savoir ce qu'il serait possible de faire.

Reviewer 2: ```suggestion
Nous décidons, en septembre 2020, d'organiser une conférence en ligne et non en présentiel, car nous savons que l'incertitude est trop démotivante pour l'équipe. Nous ne voulions pas nous lancer dans une édition "comme d'habitude mais en ligne", car nous-même n'aurions pas envie d'y participer. Nous avons plutôt imaginé **des sessions courtes, en fin de soirée, pour que les spectateurs puissent se retrouver** ensemble boire un verre de jus de pomme (ou autre) et grignoter une crêpe (ou autre), si cela pouvait être possible. Rappelez-vous, en septembre 2020 : c'était difficile de savoir ce qu'il serait possible de faire.
```

Voyons les choses du côté positif :

- Profitons-en pour inviter des speakers pour qui il est difficile de se déplacer habituellement, 
- c'est bête à dire, mais c'est toujours "mieux" que "no MiXiT"

## Aider les speakers à proposer une conférence de chez eux, de bonne qualité

Nous avons proposé diverses options pour nos speakers : 
- une conférence en live "classique" : partage d'écran, caméra et audio. Pour les aider, nous leur proposons de leur offrir du matériel (micro, caméra)

> - une conférence pré-enregistrée, et s'ils le souhaitent, nous aurions envoyer chez eux un professionnel pour la captation > cette option n'a pas été retenue par nos invités. Ceux qui ont souhaité pré-enregistrer leur conférence l'ont fait de leur propre moyen et se sont ensuite rendus disponible pour les questions/réponse (merci à eux!). Pour le cas d'une conférencière qui a du partir à l'étranger plus vite que prévu, Agnès et elle ont enregistré une session de question/réponse entre elles, avec des questions qu'Agnès avait préparé. 

Reviewer 2: ```suggestion
- une conférence pré-enregistrée, et s'ils le souhaitent, nous aurions envoyé chez eux un·e professionnel·le pour la captation > cette option n'a pas été retenue par nos invité·e·s. Ceux qui ont souhaité pré-enregistrer leur conférence l'ont fait par leurs propres moyens et se sont ensuite rendus disponible pour les questions/réponses (merci à eux !). Pour le cas d'une conférencière qui a du partir à l'étranger plus vite que prévu, Agnès et elle ont enregistré une session de questions/réponses entre elles, avec des questions préparées par Agnès. 
```

(c'est qui Agnès ? faut peut-être l'introduire ?)

Reviewer 3: "une conférencière qui a `dû` partir" ? Je suis pas 100% sûre de moi mais https://www.projet-voltaire.fr/regles-orthographe/du-ou-du/

> - une conférence pré-enregistrée, et s'ils le souhaitent, nous aurions envoyé chez eux un·e professionnel·le pour la captation > cette option n'a pas été retenue par nos invité·e·s. Ceux qui ont souhaité pré-enregistrer leur conférence l'ont fait par leurs propres moyens et se sont ensuite rendus disponible pour les questions/réponses (merci à eux !). Pour le cas d'une conférencière qui a du partir à l'étranger plus vite que prévu, Agnès et elle ont enregistré une session de questions/réponses entre elles, avec des questions préparées par Agnès. 

Reviewer 3: "une conférencière qui a dû partir" ? Je suis pas 100% sûre de moi mais https://www.projet-voltaire.fr/regles-orthographe/du-ou-du/

Author: je résiste pas au projet voltaire

<figure>
    <img src="images/posts/2021/mixit/conference_esther.jpg">
    <figcaption>
      <span class="figure__legend">Setup : partage d'écran, conférencière et micro fluffy.</span>
    </figcaption>
</figure>

## Des talks, d'accord, mais le reste ? 🥞 ??

> 2021 arrive et nous découvrons WorkAdventure, un espace virtuel où chacun peut se déplacer librement avec son avatar et rentrer en interaction avec d'autres participants, déclencher l'ouverture d'iframe pour voir des vidéos ou des sites web ou encore participer à des petites conférences grâce à Jitsi. Nous testons ici chez Elao de créer une carte de notre bureau. Résultat : c'est sympa, on prend vite l'outil en main et je découvre avec beaucoup de satisfaction la création de map 😎. 

Reviewer 2: lien vers WA ?

> 2021 arrive et nous découvrons WorkAdventure, un espace virtuel où chacun peut se déplacer librement avec son avatar et rentrer en interaction avec d'autres participants, déclencher l'ouverture d'iframe pour voir des vidéos ou des sites web ou encore participer à des petites conférences grâce à Jitsi. Nous testons ici chez Elao de créer une carte de notre bureau. Résultat : c'est sympa, on prend vite l'outil en main et je découvre avec beaucoup de satisfaction la création de map 😎. 

Reviewer 3: ```suggestion
2021 arrive et nous découvrons WorkAdventure, un espace virtuel où chacun peut se déplacer librement avec son avatar et rentrer en interaction avec d'autres participants, déclencher l'ouverture d'`iframe` pour voir des vidéos ou des sites web ou encore participer à des petites conférences grâce à Jitsi. Nous testons ici chez Elao de créer une carte de notre bureau. Résultat : c'est sympa, on prend vite l'outil en main et je découvre avec beaucoup de satisfaction la création de map 😎. 
```

Hop, la solution est vite acceptée par la team MiXiT, et nous voilà à proposer un espace virtuel aux participants et des stands aux sponsors et associations qui ont répondu à l'appel. 

<figure>
    <img src="images/posts/2021/mixit/wa-mixit.png">
    <figcaption>
      <span class="figure__legend">L'espace MiXiT dans WorkAdventure</span>
    </figcaption>
</figure>

C'est peut-être le meilleur souvenir que j'ai de la conférence, car nous nous sommes vraiment tous amusés à créer des petits mondes pour rendre la conférence plus humaine ! Bonus : ma fille a même participé en proposant des petites chèvres qui se baladent dans les mondes, et des requins dans l'océan qui borde deux des maps. Comment résister 🐐 🦈 ? Et puis j'ai passé des heures à construire avec elle des cartes pleines de petits poneys 🥰 (et de lacs de magmas). 

> Et puis surtout, la plus belle surprise de cette édition, ce sont les inventions très marrantes de Grégory, agiliste de profession, qui s'est révélé être un très bon comédien. Il nous a régalé avec un quizz très marrant qui porte sur l'histoire de MiXiT et sur de superbes vidéos d'intro. Je ne peux pas m'empêcher d'en partager ici ;

Reviewer 2: ```suggestion
Et puis surtout, la plus belle surprise de cette édition, ce sont les inventions très marrantes de Grégory, agiliste de profession, qui s'est révélé être un très bon comédien. Il nous a régalé avec un quiz très marrant qui porte sur l'histoire de MiXiT et sur de superbes vidéos d'intro. Je ne peux pas m'empêcher d'en partager ici ;
```

<figure>
    <img src="images/posts/2021/mixit/quizz-mixit.jpg">
    <figcaption>

>       <span class="figure__legend">Quizz MiXiT</span>

Reviewer 2: ```suggestion
      <span class="figure__legend">Quiz MiXiT</span>
```

    </figcaption>
</figure>

<figure>
    <img src="images/posts/2021/mixit/quizz-mixit2.jpg">
    <figcaption>

>       <span class="figure__legend">Quizz MiXiT avec question (avec la réponse D)</span>

Reviewer 2: ```suggestion
      <span class="figure__legend">Quiz MiXiT avec question (avec la réponse D)</span>
```

    </figcaption>
</figure>

## L'outil de LiveStream : Vimeo (ne faites surtout pas ça chez vous)

Bon, comment ça s'organise, une conférence en ligne ? Parce qu'en fait, c'est un peu comme d'habitude (des participants et participantes, des speakers, des sponsors), mais pas tout à fait vraiment non plus. 
Déjà, il faut un outil de livestream. Après avoir fait un petit tour du marché, on opte pour Vimeo. Nous étions déjà client pour l'hébergement du replay des conférences, et sur le papier, tout semblait coller : 

- Module de question / réponse
- Chat et modération du chat 🐈
- Possibilité de streamer nos propres vidéos, de faire des transitions sympas, d'utiliser les flux vidéo et son de la team et des invités via OBS 
- Une licence abordable pour les finances de l'association 💸

> Vimeo nous inspire confiance, nous faisons des tests où notre "seul" problème est un petit temps entre le temps réel et la transmission, qui nous semble surmontable. Hervé, un membre de l'orga, monte rapidement et avec beaucoup de talent sur l'outil (bravo à lui 👏) et prend en main la diffusion. Bon, si vous avez participé à la conférence, vous savez déjà que cela ne s'est PAS passé comme prévu 😅. 

Reviewer 2: ```suggestion
Vimeo nous inspire confiance, nous faisons des tests où notre "seul" problème est un petit lag entre le temps réel et la transmission, qui nous semble surmontable. Hervé, un membre de l'orga, monte rapidement et avec beaucoup de talent sur l'outil (bravo à lui 👏) et prend en main la diffusion. Bon, si vous avez participé à la conférence, vous savez déjà que cela ne s'est PAS passé comme prévu 😅. 
```

Le premier jour de conférence s'est déroulé avec un léger freeze de quelques minutes, magnifiquement rattrapé par Guillaume Pitron. Mais le jour suivant, le live a été interrompu de nombreuses et longues minutes, ce qui nous a amené à ouvrir une cellule de crise et à changer notre solution technique en plein milieu de la conférence, un scenario cauchemardesque comme vous pouvez l'imaginer. 

![](images/posts/2021/mixit/crazy.gif)

## La crise

Après le semi-fiasco de la soirée de mercredi soir (il y a eu au moins 20 minutes de retard pour le talk d'Isabelle Collet, la deuxième conférence de la soirée), nous décidons d'abandonner Vimeo car nous n'avons plus confiance. Fort heureusement, les suggestions d'outils et propositions n'ont pas manqué de pleuvoir auprès des membres de l'organisation. Merci à vous tous <3. 

Grâce à ça, nous basculons sur un outil open source Big Blue Button, hébergé localement par l'Ecole des Mines de Saint-Etienne. Un grand merci à eux de nous avoir fait cette proposition et de nous avoir rappelé que des solutions (qui marchent) existent, en dehors des noms déjà bien (trop) connus. 

> Aucun problème pour mettre à jour le site et faire en sorte que ce soit transparent pour nos participants (Merci CleverCloud pour l'hébergement et le déploiement zero downtime!).

Reviewer 3: ```suggestion
Aucun problème pour mettre à jour le site et faire en sorte que ce soit transparent pour nos participants (merci CleverCloud pour l'hébergement et le déploiement zero downtime !).
```

<figure>
    <img src="images/posts/2021/mixit/mixit-freeze.png">
    <figcaption>
      <span class="figure__legend">😬😬😬🥶</span>
    </figcaption>
</figure>

## Et puis l'équipe ? 

> On va pas se mentir, organiser une conférence en ligne, c'est pas aussi fun que d'habitude. Par exemple, on parle pas de la soirée, qui est le point culminant de la conférence. On ne parle pas non plus de "où on va mettre le camion à crèpes ?" et de tous les petits détails sympathiques qui vont nous rappeler que bientôt, on se voit tous, et que ça va être très cool. Et enfin, on a pas de sponsor bière (même si le sponsor bière était quand même sponsor, bisous Braincube (et vive l'Auvergne!)). 

Reviewer 2: ```suggestion
On ne va pas se mentir, organiser une conférence en ligne, c'est pas aussi fun que d'habitude. Par exemple, on ne parle pas de la soirée, qui est le point culminant de la conférence. On ne parle pas non plus de "où on va mettre le camion à crêpes ?" et de tous les petits détails sympathiques qui vont nous rappeler que bientôt, on se voit tous, et que ça va être très cool. Et enfin, on a pas de sponsor bière (même si le sponsor bière était quand même sponsor, bisous Braincube (et vive l'Auvergne!)). 
```

Bref, on a même l'impression à un moment donné qu'on a "que" les mauvais côtés de la conférence : les réunions de l'équipe se font en visio, pareil pour les discussions avec les sponsors ou avec les associations qui ont un stand sur Workadventure, on ne voit personne. Donc ça, c'est ce qu'on vit tous depuis plusieurs mois : ça nous manque. 

> Puisqu'il paraît que ça fait du bien au cerveau, voici tout de même un effet de bord positif : on a pu faire des réunions avec des gens au 4 coins ~~du monde~~ de l'Europe (Pay-Bas, Montpellier, Saint-Etienne!) et accueillir de nouveaux membres et leur chat fou (coucou Audrey 😘!). On a été un peu plus organisé sur les outils de suivi et de travail, puisqu'on avait pas vraiment le choix ! 

Reviewer 2: ```suggestion
Puisqu'il paraît que ça fait du bien au cerveau, voici tout de même un effet de bord positif : on a pu faire des réunions avec des gens au 4 coins ~~du monde~~ de l'Europe (Pays-Bas, Montpellier, Saint-Etienne !) et accueillir de nouveaux membres et leur chat fou (coucou Audrey 😘 !). On a été un peu plus organisé sur les outils de suivi et de travail, puisqu'on n'avait pas vraiment le choix ! 
```

Reviewer 3: - aux 4 coins du monde au pluriel
- on a été un peu organisés au pluriel aussi ("on" vaut "nous")

> Puisqu'il paraît que ça fait du bien au cerveau, voici tout de même un effet de bord positif : on a pu faire des réunions avec des gens au 4 coins ~~du monde~~ de l'Europe (Pays-Bas, Montpellier, Saint-Etienne !) et accueillir de nouveaux membres et leur chat fou (coucou Audrey 😘 !). On a été un peu plus organisé sur les outils de suivi et de travail, puisqu'on n'avait pas vraiment le choix ! 

Reviewer 3: ```suggestion
Puisqu'il paraît que ça fait du bien au cerveau, voici tout de même un effet de bord positif : on a pu faire des réunions avec des gens aux 4 coins ~~du monde~~ de l'Europe (Pays-Bas, Montpellier, Saint-Etienne !) et accueillir de nouveaux membres et leur chat fou (coucou Audrey 😘 !). On a été un peu plus organisés sur les outils de suivi et de travail, puisqu'on n'avait pas vraiment le choix ! 
```

## Et maintenant ?

> Certes, la réalisation de cette édition n'a pas toujours été facile. Il y a toujours des aléas, mais le freeze du live a vraiment été un gros coup porté au moral. Mais le soutien de nos sponsors et la bienveillance des participant(e)s nous a fait chaud au coeur, et a rendu la conférence plus chaleureuse que prévu. Merci, vraiment, notamment à Florence C. des Duchesses et Esther, notre invitée géniale ! 

Reviewer 2: ```suggestion
Certes, la réalisation de cette édition n'a pas toujours été facile. Il y a toujours des aléas, mais le freeze du live a vraiment été un gros coup porté au moral. Heureusement, le soutien de nos sponsors et la bienveillance des participant·e·s nous a fait chaud au coeur, et a rendu la conférence plus chaleureuse que prévu. Merci, vraiment, notamment à Florence C. des Duchesses et Esther, notre invitée géniale ! 
```

---
title:              "MiXiT 2021, une conférence 100 % en ligne : retour d'expérience côté organisation"
date:               "2021-05-25"
lastModified:       ~
tableOfContent:     2

description:        "Comme la plupart des conférences tech, cette année, l'édition de MiXiT 2021 a eu lieu à distance. Alors, comment ça s'est passé côté orga ?"

thumbnail:          images/posts/2021/teambuilding/workadventure.png
tags:               ["conference", "mixit"]
categories:         ["Elao"]
authors:            ["aldeboissieu"]
---

> Quand Agnès Crépet m'a proposé de rejoindre l'aventure MiXiT en 2018, je dois reconnaître que je n'ai pas hésité très longtemps. D'abord parce que j'avais eu une excellente expérience de la conférence - j'avais réussi à obtenir une place au tirage au sort dès ma première inscription (😎), et puisque que j'ai une certaine appétence pour le milieu associatif et que j'ai senti que je pouvais bien m'entendre avec l'équipe en place. 

Author: 😇

Le challenge de l'édition 20219, c'était le changement de lieu pour la Manufacture de Tabacs, dans le but d'accueillir plus participants : de 350 à plus de 1000. Puis, vient 2020. L'organisation se passe bien, le programme est ficelé. Mais comme vous le savez, la mort dans l'âme, nous avons dû annuler car l'incertitude de la suite pesait trop sur notre moral. Finalement, nous avons bien fait, puisque nous avons été ensuite confinés par la suite.

> Pourquoi faire un retour d'expérience cette année, et pas les années précédentes ? Et bien tout simplement parce que cette année n'a pas été comme les autres sur l'organisation même de la conférence et celle de l'équipe. En passant, je tiens à remercier mon employeur Elao (qui héberge d'ailleurs ce billet de blog), qui permet à ses collaborateurs (et donc à moi) de prendre du temps de travail pour contribuer à des projets open source ou à des projets comme MiXiT, en estimant que c'est une façon de redistribuer ce dont on profite. Si vous avez une activité bénévole, vous devez vous douter que c'est très confortable de pouvoir prendre 20 minutes de temps en temps dans sa journée (en dehors de ses pauses) pour répondre à un sponsor, envoyer un mail à un speaker ...

Reviewer: ```suggestion
Pourquoi faire un retour d'expérience cette année, et pas les années précédentes ? Et bien tout simplement parce que cette année n'a pas été comme les autres sur l'organisation même de la conférence et celle de l'équipe. En passant, je tiens à remercier mon employeur Elao (qui héberge d'ailleurs ce billet de blog), qui permet à ses collaborateurs (et donc à moi) de prendre du temps de travail pour contribuer à des projets open source ou à des projets comme MiXiT, en estimant que c'est une façon de redistribuer ce dont on profite. Si vous avez une activité bénévole, vous devez vous douter que c'est très confortable de pouvoir prendre 20 minutes de temps en temps dans sa journée (en dehors de ses pauses) pour répondre à un sponsor, envoyer un mail à un speaker…
```

https://elao.github.io/elao_/blog/styleguide/example/#ponctuation

## Le choix de la conférence en ligne

> Nous décidons, en septembre 2020, d'organiser une conférence en ligne et non en présence, car nous savons que l'incertitude est trop démotivante pour l'équipe. Nous ne voulions pas nous lancer dans une édition "comme d'habitude mais en ligne", car nous-même n'aurions pas envie d'y participer. Nous avons plutôt imaginé des sessions courtes, en fin de soirée, pour que les spectateurs puissent se retrouver ensemble boire un verre de jus de pomme (ou autre) et grignoter une crêpe (ou autre), si cela pouvait être possible. Rappelez-vous, en septembre 2020, c'était difficile de savoir ce qu'il serait possible de faire.'

Reviewer: ```suggestion
Nous décidons, en septembre 2020, d'organiser une conférence en ligne et non en présentiel, car nous savons que l'incertitude est trop démotivante pour l'équipe. Nous ne voulions pas nous lancer dans une édition "comme d'habitude mais en ligne", car nous-même n'aurions pas envie d'y participer. Nous avons plutôt imaginé des sessions courtes, en fin de soirée, pour que les spectateurs puissent se retrouver ensemble, boire un verre de jus de pomme (ou autre) et grignoter une crêpe (ou autre), si cela pouvait être possible. Rappelez-vous, en septembre 2020: c'était difficile de savoir ce qu'il serait possible de faire.
```

?

Author: ouais c'est plus joli ! merci

## Des talks, d'accord, mais le reste ? 🥞 ??

2021 arrive et nous découvrons WorkAdventure, un espace virtuel où chacun peut se déplacer librement avec son avatar et rentrer en interaction avec d'autres participants, déclencher l'ouverture d'iframe pour voir des vidéos ou des sites web ou encore participer à des petites conférences grâce à Jitsi. Nous testons ici chez Elao de créer une carte de notre bureau. Résultat : c'est sympa, on prend vite l'outil en main et je découvre avec beaucoup de satisfaction la création de map 😎. Hop, la solution est vite acceptée par la team MiXiT, et nous voilà à proposer un espace virtuel aux participants et des stands aux sponsors et associations qui ont répondu à l'appel. 
C'est peut-être le meilleur souvenir que j'ai de la conférence, car nous nous sommes vraiment tous amusés à créer des petits mondes pour rendre la conférence plus humaine ! Bonus : ma fille a même participé en proposant des petites chèvres qui se baladent dans les mondes, et des requins dans l'océan qui borde deux des maps. Comment résister 🐐 🦈 ? Et puis j'ai passé des heures à construire avec elle des cartes pleines de petits poneys 🥰 (et de lacs de magmas). 

> Et puis surtout, la plus belle surprise de cette édition, ce sont les inventions très marrantes de Grégory, agiliste de profession, qui s'est révélé être un très bon comédien. Il nous a régalé avec un quizz très marrant qui porte sur l'histoire de MiXiT et sur de superbes vidéos d'intro. Je ne peux pas m'empêcher d'en partager ici ;

Reviewer: > Je ne peux pas m'empêcher d'en partager ici ;

Il manque la suite ? 😄

Author: Ouais, ça sera des vidéos :)

Puisqu'il paraît que ça fait du bien au cerveau, voici tout de même les effets positifs : 
- On a pu faire des réunions avec des gens au 4 coins ~~du monde~~ de l'Europe (Pay-Bas, Montpellier, Saint-Etienne!) et accueillir de nouveaux membres et leur chat fou (coucou Audrey 😘!). On a été un peu plus organisé sur les outils de suivi et de travail, puisqu'on avait pas vraiment le choix ! 

## Et maintenant ?

Certes, la réalisation de cette édition n'a pas toujours été facile. Il y a toujours des aléas, mais le freeze du live a vraiment été un gros coup porté au moral. Mais le soutien de nos sponsors et la bienveillance des participant(e)s nous a fait chaud au coeur, et a rendu la conférence plus chaleureuse que prévu. Merci, vraiment, notamment à Florence C. des Duchesses et Esther, notre invitée géniale ! 

> De cette édition, je retiendrai surtout la programmation vraiment exceptionnelle (le replay est disponible), que nous avons reçu un très beau soutien de notre communauté, et enfin que les membres de votre équipe de bénévole peut avoir des talents vraiment insoupçonnés. 

Reviewer: ```suggestion
De cette édition, je retiendrai surtout la programmation vraiment exceptionnelle (le replay est disponible), que nous avons reçu un très beau soutien de notre communauté, et enfin que les membres de votre équipe de bénévoles peut avoir des talents vraiment insoupçonnés. 
```
